.github/workflows/cache-dependencies.yml

```diff
@@ -52,3 +55,43 @@ jobs:
 
       - name: Check packages
         run: conda list
+  cache-doc-dependencies:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v2
+
+## DOCUMENTATION CACHE ##
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge # mamba is faster than base conda
+          miniforge-version: latest
+          activate-environment: saltproc-doc-env
```

Comment:
Not particularly. I still think it's good in general to decouple the docs from the testing since trying to build the docs when the `saltproc` package is installed causes a bunch of warnings to pop up due to duplicate files.